convert2rhel/unit_tests/backup_test.py

```diff
@@ -579,28 +563,28 @@ def test_restore_uninstall(self, run_subprocess_with_empty_rpmdb, rpm_key):
 
         # Check that the beginning of the run_subprocess call starts with the command to remove
         # the key (The arguments our fixture has added to use the empty rpmdb come after that)
-        assert run_subprocess_with_empty_rpmdb.called_with[-1][0][0][0:3] == ["rpm", "-e", "gpg-pubkey-fd431d51"]
+        assert run_subprocess_with_empty_rpmdb.call_args_list[-1][0][0][0:3] == ["rpm", "-e", "gpg-pubkey-fd431d51"]
```

Comment:
This before and after is quite confusing with the array fetching, the `[-1][0][0][0:3]` is difficult to parse all at once

But nothing to do as part of this PR